x-pack/plugin/watcher/src/test/java/org/elasticsearch/xpack/watcher/common/http/HttpClientTests.java

```diff
@@ -167,88 +167,67 @@ public void testHttps() throws Exception {
         Path certPath = getDataPath("/org/elasticsearch/xpack/security/keystore/testnode.crt");
         Path keyPath = getDataPath("/org/elasticsearch/xpack/security/keystore/testnode.pem");
         MockSecureSettings secureSettings = new MockSecureSettings();
-        Settings settings;
-        if (randomBoolean()) {
-            settings = Settings.builder()
-                .put("xpack.http.ssl.certificate_authorities", trustedCertPath)
-                .setSecureSettings(secureSettings)
-                .build();
-        } else {
-            settings = Settings.builder()
-                .put("xpack.ssl.certificate_authorities", trustedCertPath)
-                .setSecureSettings(secureSettings)
-                .build();
-        }
+        Settings settings = Settings.builder()
+            .put("xpack.http.ssl.certificate_authorities", trustedCertPath)
+            .setSecureSettings(secureSettings)
+            .build();
         try (HttpClient client = new HttpClient(settings, new SSLService(settings, environment), null)) {
             secureSettings = new MockSecureSettings();
             // We can't use the client created above for the server since it is only a truststore
-            secureSettings.setString("xpack.ssl.secure_key_passphrase", "testnode");
+            secureSettings.setString("xpack.security.transport.ssl.secure_key_passphrase", "testnode");
             Settings settings2 = Settings.builder()
-                .put("xpack.ssl.key", keyPath)
-                .put("xpack.ssl.certificate", certPath)
+                .put("xpack.security.transport.ssl.key", keyPath)
```

Comment:
same as above, just for being easy to the eye use the `http` layer settings for the MockWebServer